components/log-viewer-webui/server/src/app.test.js

```diff
@@ -2,25 +2,48 @@ import httpStatusCodes from "http-status-codes";
 import {test} from "tap";
 
 import app from "./app.js";
+import {parseEnvVars} from "./utils.js";
 
 
 test("Tests the example routes", async (t) => {
-    const server = await app();
+    const envVars = parseEnvVars();
```

Comment:
We might want to update the [docs](https://docs.yscope.com/clp/main/dev-guide/components-log-viewer-webui.html#testing) to state the CLP package / testing DB servers should be up before these tests can be run.